django_cloud_tasks/tasks/task.py

```diff
@@ -148,4 +150,7 @@ def url(cls):
 
     @property
     def __client(self):
-        return CloudTasks()
+        if not self.__class__._client:
+            self.__class__._client = CloudTasks()
+
+        return self.__class__._client
```

Comment:
This got a little weird, forcing the instance to set the class attribute.
Maybe changing this to a class method would make it cleaner, what do you think?